assets/react/v3/shared/utils/style-utils.ts

```diff
@@ -485,4 +488,109 @@ export const styleUtils = {
       outline-offset: 1px;
     }
   `,
+  optionCheckButton: css`
```

Comment:
How about making an atom?

Reply:
Wouldn't it be overkill, `optionCheckButton` is only used twice in this whole project?